worker/src/main/scala/org/apache/celeborn/service/deploy/worker/WorkerSource.scala

```diff
@@ -72,6 +73,8 @@ class WorkerSource(conf: CelebornConf) extends AbstractSource(conf, MetricsSyste
   addTimer(REPLICA_REGION_START_TIME)
   addTimer(PRIMARY_REGION_FINISH_TIME)
   addTimer(REPLICA_REGION_FINISH_TIME)
+  addTimer(PRIMARY_SEGMENT_START_TIME)
```

Comment:
The introduced metrics should be added in document and `celeborn-dashboard.json`.